README.md

```diff
@@ -30,6 +30,27 @@ Current contributors include:
 
 [![Figure 1. _iSEE_ uses a customisable multi-panel layout.][Figure1]](https://f1000research.com/articles/7-741/v1)
 
+## Installation
+
+_iSEE_ can be easily installed from Bioconductor using `BiocManager::install()`:
+
+``` r
+if (!requireNamespace("BiocManager", quietly = TRUE))
+    install.packages("BiocManager")
+BiocManager::install("iSEE")
+```
+
+Optionally, if you want to install the development version from GitHub, you can use:
```

Comment:
Why are we encouraging this?

Reply:
I was split for that
- on one side, it is the GitHub version, so maybe someone would want it (and would like to see the instructions)
- OTOH, advanced users could know the drill anyway
Should this just be gone?

Reply:
Yes.

Reply:
Ok, going away then.